### PR TITLE
Add '--wait' flag to partitions.js

### PR DESCRIPTION
**Pull request by Motiejus**
This will keep a connection to ringpops open.

### CODE REVIEW

File: partitions.js [hunk 1]
```diff
@@ -51,10 +72,10 @@ function main() {
                 'Sample Host'
             ];
         }
-
```
Comment (CorgiMan) [inline partitions.js]: why remove this line?

Comment (Motiejus) [inline partitions.js]: Accident. Will bring it back.

**Comment by thanodnl** [comment]
LGTM

**Comment by CorgiMan** [comment]
Looks good!

**Comment by Motiejus** [comment]
Squashed the commits. I`ll merge as soon as travis-ci check is done.

**Pull request merged**